lighthouse-core/lib/web-inspector.js

```diff
@@ -30,10 +30,10 @@ module.exports = (function() {
   const _setImmediate = global.setImmediate;
 
   global.Runtime = global.Runtime || {};
-
-  // Required for devtools-timeline-model
   global.Runtime.experiments = global.Runtime.experiments || {};
-  global.Runtime.experiments.isEnabled = global.Runtime.experiments.isEnabled || (_ => false);
+  // DevTools runtime doesn't know about some experiments that DTM looks for
+  // To avoid exceptions, we assume all experiments are disabled
+  global.Runtime.experiments.isEnabled = (_ => false);
```

Comment:
the fix we needed last time is just that this function existed? or did we really need to use the built-in?

Reply:
seems like we just needed this function to exist before https://github.com/GoogleChrome/lighthouse/pull/3497/files#diff-daa5294aa1f14b31b082e6155d79f2d3L29